core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/UserDevfile.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.core.model.workspace.devfile;
+
+/** Devfile that persisted in permanent storage. */
+public interface UserDevfile {
+  /** Returns the identifier of this persisted devfile instance. It is mandatory and unique. */
+  String getId();
+
+  /** Returns the name of devfile. It is mandatory. */
+  String getName();
+
+  /**
+   * Returns the namespace of the current devfile instance. Devfile name is unique for devfiles in
+   * the same namespace.
+   */
+  String getNamespace();
```

Comment:
I know the "namespace" here means "user or organization" but I am not sure we should stick to this Che6 nomenclature. It is highly confusing to use the same name for completely different things in the model and in the infrastructures.

What about calling this `ownerName` or `accountName`? It would IMHO slightly better express where this is coming from as well as imply the same semantics (at least to an extent) - the same owner cannot have two devfiles with the same name, yet devfiles with the same name can have different owners.

Reply:
aha, that's what baffled me (https://github.com/eclipse/che/pull/17843#discussion_r490924937) :D